### PR TITLE
included tutorial on lora nvram usage

### PR DESCRIPTION
**Pull request by gijsio**

### CODE REVIEW

File: content/tutorials/networks/lora/nvram.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+sleep_time = 1000
+print("init LoRa")
+lora = LoRa(mode=LoRa.LORAWAN, region=LoRa.EU868)
+for i in range(0,10):
+    time.sleep(0.1) #be able to ctrl+c out of this
+# create an ABP authentication params
+dev_addr = struct.unpack(">l", ubinascii.unhexlify(''))[0]
+nwk_swkey = ubinascii.unhexlify('')
+app_swkey = ubinascii.unhexlify('')
```
Comment (peter-pycom) [inline content/tutorials/networks/lora/nvram.md]: do we need this initialization with unhexlify('') ? it looks strange

File: content/tutorials/networks/lora/nvram.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import time
+from network import LoRa
+import socket
+import ubinascii
+
+sleep_time = 1000
+print("init LoRa")
+lora = LoRa(mode=LoRa.LORAWAN, region=LoRa.EU868)
+for i in range(0,10):
+    time.sleep(0.1) #be able to ctrl+c out of this
```
Comment (peter-pycom) [inline content/tutorials/networks/lora/nvram.md]: why would anyone want to ctrl-c out of this 1 sec sleep in the middle of the example? I would keep it simple and just 

time.sleep(1)

even if someone want's to ctrl c this example, they can do right before or right after this second.

what would be nice is to comment on why we need the second in the first place

**Review by peter-pycom — CHANGES_REQUESTED** [review]

File: content/tutorials/networks/lora/nvram.md (hunk 2 above)
Comment (gijsio) [reply content/tutorials/networks/lora/nvram.md]: Ill change it 👍

File: content/tutorials/networks/lora/nvram.md (hunk 1 above)
Comment (gijsio) [reply content/tutorials/networks/lora/nvram.md]: We always do it like this for the LoRa join parameters. This way you can copy the TTN values into the code

Comment (peter-pycom) [reply content/tutorials/networks/lora/nvram.md]: ah, now I get it. I am supposed to fill this in to make it work (I have never tried abp before)

why not keep it consistent with the other abp example? 

https://docs.pycom.io/tutorials/networks/lora/lorawan-abp/

Comment (gijsio) [reply content/tutorials/networks/lora/nvram.md]: Yes, we can :)

**Review by peter-pycom — APPROVED** [review]

**Pull request merged**